elements/pl-dropdown/pl-dropdown.mustache

```diff
@@ -32,12 +32,14 @@
         {{^parse-error}}
             <code class="user-output">{{{submitted-answer}}}</code>
 
+            {{#display-score-badge}}
             {{#correct}}
```

Comment:
This subsequent block should be indented.

Reply:
Your wish has been granted...